New Game:-Dice_Rolling_simulator

priyashuu opened the pull request:
## PR Description 📜
**** Description
The dice rolling simulator is a simple application built using Python and the Tkinter library for the graphical user interface (GUI). The simulator mimics the action of rolling a dice, providing a random outcome between 1 and 6 each time the user interacts with the interface. This application is useful for games, educational purposes, or simply for fun.

**Game Logic**

Initialization: When the program starts, it initializes the main window using Tkinter.
Dice Roll: A function generates a random number between 1 and 6 using Python's random module.
Display: The result of the dice roll is displayed on the GUI, typically as a number or a graphical representation of a dice face.
Interaction: The user can roll the dice by clicking a button on the GUI, which triggers the dice roll function.
Fixes # <4754>

<hr>
 
## Mark the task you have completed ✅



- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.


<hr>

## Add your screenshots(Optional) 📸

https://github.com/kunjgit/GameZone/assets/150767072/09aca835-566a-4c15-8e72-d88985316bec

![Screenshot 2024-07-07 161502](https://github.com/kunjgit/GameZone/assets/150767072/c962eed6-3d28-4a62-8ba0-db3eb283868b)




--- 
<br>

## Thank you soo much for contributing to our repository 💗

kunjgit commented:
Thank you @priyashuu ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

ishita-43 reviewed (changes requested):
Hi please do the following changes:- 
add your game entry in readme.md file of the repo this will make changes made by you visible in the live project 
and also add a image of your game  in asset folder of the repo follow naming convention while  naming the image mentioned in contribution guidelines

priyashuu commented:
Hi  @ishita-43 ,
 I wanted to inform you that I accidentally renamed my branch during my last commit, which seems to have caused a conflict with the current pull request. 

1. I renamed the branch by mistake.
2. I have now reverted the branch name back to its original name (or I have opened a new pull request with the correct branch name if renaming wasn't possible).
3. I have fetched the latest changes from the base branch and resolved any merge conflicts.

Could you please review the updated pull request and let me know if any further actions are required from my side?

ishita-43 commented on README.md:
```diff
@@ -844,6 +844,12 @@ This repository also provides one such platforms where contributers come over an
 | [Tic_Tac_Toe_Neon](https://github.com/kunjgit/GameZone/tree/main/Games/Tic_Tac_Toe_Neon) | [Magic_8_ball_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Magic_8_ball) |
 | [Catch_Craze](https://github.com/kunjgit/GameZone/tree/main/Games/Catch_Craze) |
+<<<<<<< HEAD
+| [Dice_Rolling_Simulator](https://github.com/priyashuu/GameZone/tree/main/Games/Dice_rolling_simulator)
```
remove the duplicate entry

ishita-43 commented on README.md:
```diff
@@ -844,6 +844,12 @@ This repository also provides one such platforms where contributers come over an
 | [Catch_Craze](https://github.com/kunjgit/GameZone/tree/main/Games/Catch_Craze) |
+<<<<<<< HEAD
+| [Dice_Rolling_Simulator](https://github.com/priyashuu/GameZone/tree/main/Games/Dice_rolling_simulator)
+=======
```
remove this line

ishita-43 commented on README.md:
```diff
@@ -844,6 +844,12 @@ This repository also provides one such platforms where contributers come over an
+=======
+|[Dice_Rolling_Simulator](https://github.com/priyashuu/GameZone/tree/priyashuu-patch-1/Games/Dice_rolling_simulator)
+
+>>>>>>> priyashuu-patch-1
```
remove this line also
everything else is good

ishita-43 reviewed (approved):
Approved!

kunjgit commented:
Thank you  @priyashuu , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

priyashuu commented:
Thank you so much for your kind words❤️ .
I just want to ask why my game level points got reduced, I just want to ask for improvement in future..
See you soon with a next PR🫡
